spec/services/mark_inactive_application_spec.rb

```diff
@@ -11,7 +11,8 @@
     end
 
     it 'updates inactive at field', time: Time.zone.local(2023, 11, 10) do
-      expect { mark_inactive_application.call }.to change(application_choice, :inactive_at).from(nil).to(Time.zone.local(2023, 11, 10))
```

Comment:
Was this spec not passing?

Reply:
On my machine, it failed 1 out of 3 (before the change). Not sure why (because transactions should commit). So I decided to change.